data/objects/library_book_view_controller.cfg

```diff
@@ -143,11 +143,11 @@
 					   set(value.alpha, 255)
 					),
 
-					if(controller.showing_shop and entry.num_copies < 3, [
+					if(controller.showing_shop and entry.num_copies < 4, [
```

Comment:
This is change would arguably be questionable if accompanied by sibling change in `data/objects/library_scroll_view_controller.cfg`.

Without it, it seems really suspicious.

Are you really sure you want to add this change?

Reply:
It has to be changed. Because unlike scroll view the book view shows 4 columns of cards and not 3 columns. If the previous value is left then insufficient amount of "Buy" buttons and cost indicators is shown.
I left scroll view unchanged. If I also made it show 4 columns than the same change had to be done.

Reply:
Ok, I just checked if the change also has to be done in the scroll view config and it does. Will update the PR.